src/main/java/frc/robot/commands/CDSBallManagementCommand.java

```diff
@@ -40,7 +43,9 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
```

Comment:
Do we need to run the cargo stopper motor in reverse for this command too, so we don't overrun it?